curriculum/challenges/english/08-coding-interview-prep/rosetta-code/self-describing-numbers.english.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Self Describing Numbers
+id: 5eaf48389ee512d4d103684b
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+There are several so-called "self describing" or <a href="https://en.wikipedia.org/wiki/Self-descriptive_number">"self-descriptive"</a> integers. <br />
+
+An integer is said to be "self-describing" if it has the property that, when digit positions are labeled 0 to N-1, the digit in each postion is eqal to the number of times that that digit appears in the number. <br />
+
+For example, <b>2020</b> is a four-digit self describing number: <br />
+<ul>
+    <li> position 0 has value 2 and there are two 0s in the number; </li>
+    <li> position 1 has value 0 and there are no 1s in the number; </li>
+    <li> position 2 has value 2 and there are two 2s; </li>
+    <li> position 3 has value 0 and there are zero 3s; </li>
+</ul>
+
+Self-describing numbers < 100.000.000  are:     1210,   2020,   21200,   3211000,   42101000.
+</section>
+
+## Instructions
+<section id='instructions'>
+
+Write a function that will check whether a given positive integer is self-describing.
```

Comment:
This can be changed slightly to be more in line with some of the other instructions in the Rosetta Code section:

Write a function that takes a positive integer as a parameter. If it is self-describing return true. Otherwise, return false.